Umami/README.md

```diff
@@ -0,0 +1,42 @@
+# Install Umami on Codesphere
+Hey folks,
+Have you ever wondered whether you can host your own Google Analytics alternative (because Google collects a lot of data from it)?
+
+If yes, you’re on the very correct blog post!
+
+Let’s dive into how we can deploy our own Analytics service using Codesphere Cloud!
+
+## Steps for installing Umami on Codesphere.
+1. Signup On Codesphere — https://codesphere.com/
+2. Create a new instance using the “+ New Workspace” button.
+3. Put in the URL: https://github.com/sancho1952007/install-umami-on-codesphere.git in the input box.
+4. Choose the “Boost” plan at the minimum with Always On/Off or when unused based on your requirements (installation might crash on the other lower plans due to lack of resources/storage).
+5. Click on the “Create button”.
+6. Go to the “Setup” section on the top-left and then click on “Env vars”.
+7. Click on “+ Add new variable” and then put DATABASE_URL in the left input box.
+8. Enter your database MySQL / PostgreSQL Database URL (You can create one on [Codesphere](https://docs.codesphere.com/integrations/databases/) if you don’t have it) in the right side input.
+9. Go to the CI Pipeline at the bottom of the screen.
+10. Click on the Prepare section and then press the “Run” button.
+11. My ci.yml will do all the work for you. Wait for the “Prepare” box to end. Afterward, Go to the Run section and click the “Run” button.
+12. Once ✓ Ready is displayed in the terminal, click on the Open Deployment button on the top panel. Enter the default Umami credentials i.e., username: admin and password: umami
+
+You're ready to now use it on your site!
+
+Also, If the restart fails in the future, Follow step 9, then directly go to the Run section, and then click the "Run" button.
```

Comment:
Does the restart fail sometimes? What is the reason why the restart fails?

Reply:
Just a fallback in case of Codesphere's downtime or failure to start the ci.yml's Run command.